scss/components/_tabsMenu.scss

```diff
@@ -3,16 +3,23 @@
 	margin: 0;
 }
 
+#zotero-tabs-menu-panel {
+	min-height: 0;
```

Comment:
Can we add a comment for this?

Reply:
I rebased and it looks like this css is no longer needed. This must've been a left-over either from pre-fx115, or from my initial attempt to setup the scrollable menu without explicitly calculating `maxHeight` with JS. So I just removed it.